jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -161,11 +163,50 @@ private static PlatformParameters of(String osArchitecture) {
     }
   }
 
+  /** Configuration for {@code [from|to].credHelper} parameter. */
+  public static class CredHelperParameters implements CredHelperConfiguration {
+    @Nullable @Parameter private String helper;
+    @Parameter private Map<String, String> environment = new HashMap<>();
+
+    @Override
+    public Optional<String> getHelperName() {
+      return Optional.ofNullable(helper);
+    }
+
+    @Override
+    public Map<String, String> getEnvironment() {
+      return environment;
+    }
+
+    public void setHelper(@Nullable String helper) {
```

Comment:
The `helper` argument is nullable here but it's not in `set()`. Should these be consistent?

Reply:
Will change - they should be